.eslintrc.json

```diff
@@ -1,5 +1,14 @@
 {
-	"extends": "6river/typescript",
+	"extends": [
+		"6river/typescript",
+		"plugin:import/typescript",
```

Comment:
I don't think we have any rules from this plugin enabled, so it's effectively a no-op at the moment. I like the idea of using it, but have not taken the time to enable any rules for it in `eslint-config-6river`.

We use [these rules](https://github.com/6RiverSystems/mfp-ui/blob/1b2613e2c206753c9d3682f4ce80bb1770e0f4ed/.eslintrc.json#L30-L41) in `mfp-ui`, and only the last one would be relevant in non-Webpack projects.

Reply:
aah, I want import order enforcement, I'll turn that on here :D